main.py

```diff
@@ -63,6 +63,8 @@ def main(args):
     parser.add_argument('--slot_loss_coef', type=float, default=1.0, help='Coefficient for the slot loss.')
 
     parser.add_argument('--results_logging', type=str, help="Optional 'dev' metrics CSV file to store metrics to")
+    parser.add_argument('--best_model_metrics', type=str,
+                        help="Comma-separated list of result names (metrics) to optimize and save the model: " + ", ".join(METRICS_MAXIMIZE.keys()))
```

Comment:
This formulation is a little strange. What about:

`Comma-separated list of metrics to optimize for and save. Select values from: " + ", ".join(METRICS_MAXIMIZE.keys())` 

There's also a possibility to provide a list instead comma-separated string to argparse (via `nargs='*`), but your solution is probably better since it is consistent with the library's convention.

Reply:
You are right, using `nargs` makes sense. 
Will fix.

Reply:
Fixed.